cpp/oneapi/dal/algo/svm/common.hpp

```diff
@@ -250,42 +218,100 @@ class descriptor : public detail::descriptor_base<Task> {
         return *this;
     }
 
+    /// The upper bound $C$ in constraints of the quadratic optimization problem.
+    /// @invariant :expr:`c > 0`
+    /// @remark default = 1.0
+    double get_c() const {
+        return base_t::get_c();
+    }
+
     auto& set_c(double value) {
         base_t::set_c_impl(value);
         return *this;
     }
 
-    auto& set_accuracy_threshold(double value) {
-        base_t::set_accuracy_threshold_impl(value);
-        return *this;
+    /// The maximum number of iterations $T$
+    /// @invariant :expr:`max_iteration_count >= 0`
+    /// @remark default = 100000
+    std::int64_t get_max_iteration_count() const {
+        return base_t::get_max_iteration_count();
     }
 
     auto& set_max_iteration_count(std::int64_t value) {
         base_t::set_max_iteration_count_impl(value);
         return *this;
     }
 
+    /// The threshold $\\varepsilon$ for the stop condition
+    /// @invariant :expr:`accuracy_threshold >= 0.0`
+    /// @remark default = 0.0
+    double get_accuracy_threshold() const {
+        return base_t::get_accuracy_threshold();
+    }
+
+    auto& set_accuracy_threshold(double value) {
+        base_t::set_accuracy_threshold_impl(value);
+        return *this;
+    }
+
+    /// The size of cache in megabytes for storing values of the kernel matrix.
+    /// @invariant :expr:`cache_size >= 0.0`
+    /// @remark default = 200.0
+    double get_cache_size() const {
+        return base_t::get_cache_size();
+    }
+
     auto& set_cache_size(double value) {
         base_t::set_cache_size_impl(value);
         return *this;
     }
 
+    /// The parameter of the WSS scheme $\\tau$.
```

Comment:
what is WSS?

Reply:
@PetrovKP, probably we should not refer to WSS in API reference, the meaning is not clear from the context. Suggest better description for this param.

Reply:
I agree. I will fix